basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/BaseLTIService.java

```diff
@@ -719,10 +786,11 @@ protected String deleteContentLinkDao(Long key, String siteId, boolean isAdminRo
 					return rb.getString("error.placement.not.removed");
 				}
 			} else {
-				M_log.warn(this + " LTI content="+key+" placement="+tool.getId()+" could not find page in site=" + siteStr);
+				M_log.warn("LTI content="+key+" placement="+tool.getId()+" could not find page in site=" + siteStr);
```

Comment:
use new logging:
M_log.warn("LTI content={} placement={} could not find page in site={}", key, tool.getId(), siteStr);